Sources/APIErrors.swift

```diff
@@ -16,6 +16,7 @@ public enum OktaAPIError: Error {
     case noWellKnown
     case noJWKSEndpoint
     case noKey
+    case offline
```

Comment:
Makes sense.

But then it should cover probably all cases from `NSURLErrorDomain`, doesn't it? 

@mikenachbaur-okta What are your thoughts?

Reply:
There are other categories of error that are unrelated to the concept of "offline". I'd keep this scoped to just errors indicating the user is offline and unable to fetch the key information, because if a developer is going to use this to determine whether or not to trust the ID token without validating its key, we shouldn't lump other errors that are unrelated.

If a developer needs raw access to the error in the future, we can provide that at a later date.

Reply:
Done.